src/components/AccommodationCard/AccommodationCard.tsx

```diff
@@ -0,0 +1,28 @@
+import {CardWrapper, Figure, Image, Title, Location, Price, Star} from './AccommodationCardStyle';
+import StarImage from '../../assets/star.png';
+
+interface Data {
+    imgSrc: string;
+    title: string;
+    location: string;
+    price: number;
+    categorization: number;
+}
+
+const AccommodationCard: React.FC<Data> = ({imgSrc, title, location, price, categorization}) => {
+    return (
+        <CardWrapper>
+            <Figure>
+                <Image src={imgSrc}/>
+            </Figure>
+            <Title>{title}</Title>
+            <Location>{location}</Location>
+            <Price>EUR {price}</Price>
+            {[...Array(categorization)].map((index) => ( 
+                <Star key={index} src={StarImage}/>
+            ))}
```

Comment:
Ovdje također možeš koristiti getStars funkciju na isti način kao što sam napisala za AccomodationDetails. Samim time, budući da se getStarts koristi na već 2 mjesta, možeš napraviti datoteku naziva `helper` ili slično, u kojoj ćeš staviti tu funkciju (i exportati je da je možeš dohvaćati s drugih mjesta).
Ovako kako si trenutno napisao, dobiva se greška u consoli za key props unutar listi.